compose/config/config.py

```diff
@@ -408,7 +409,13 @@ def validate_and_construct_extends(self):
         extended_file = process_config_file(
             extends_file,
             service_name=service_name)
-        service_config = extended_file.config[service_name]
+
+        if extended_file.version == 2:
+            service_config = extended_file.config['services'][service_name]
+
+        if extended_file.version == 1:
+            service_config = extended_file.config[service_name]
```

Comment:
We'd like to keep the version if/else in as few places as possible.  Right now most of it is done in the `ConfigFile` class (this `extended_file` is an instance of one).

Could you move this condition into a `ConfigFile.get_service(name)` so this call can be:

```python

service_config = extended_file.get_service(service_name)

```